infra/config-store/src/main/java/com/simprints/infra/config/store/local/models/FaceConfiguration.kt

```diff
@@ -20,10 +35,24 @@ internal fun FaceConfiguration.ImageSavingStrategy.toProto(): ProtoFaceConfigura
 
 internal fun ProtoFaceConfiguration.toDomain(): FaceConfiguration =
     FaceConfiguration(
-        nbOfImagesToCapture,
-        qualityThreshold,
-        imageSavingStrategy.toDomain(),
-        decisionPolicy.toDomain(),
+        allowedSDKs = allowedSdksList.map { it.toDomain() },
+        if (hasRankOne()) rankOne.toDomain() else null,
+    )
+
+@Suppress("SameReturnValue")
+internal fun ProtoFaceConfiguration.ProtoBioSdk.toDomain() = when (this) {
+    ProtoFaceConfiguration.ProtoBioSdk.RANK_ONE -> FaceConfiguration.BioSdk.RANK_ONE
+    ProtoFaceConfiguration.ProtoBioSdk.UNRECOGNIZED -> FaceConfiguration.BioSdk.RANK_ONE
```

Comment:
`ProtoFaceConfiguration.ProtoBioSdk.UNRECOGNIZED -> FaceConfiguration.BioSdk.RANK_ONE`
Just making sure that this is a correct mapping

Reply:
Yes, it's correct. Kotlin wants us to always have `UNRECOGNIZED` but we only have one face SDK currently so we map to it.